vmngclient/api/task_status_api.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import List, Union
+
+from tenacity import retry, retry_if_result, stop_after_attempt, wait_fixed  # type: ignore
+
+from vmngclient.session import vManageSession
+from vmngclient.utils.creation_tools import get_logger_name
+from vmngclient.utils.operation_status import OperationStatus, OperationStatusId
+
+logger = logging.getLogger(get_logger_name(__name__))
+
+
+class TaskStatus:
+    """API class to check task status"""
+
+    def __init__(self, session: vManageSession):
+        self.session = session
+        self.status: str = ''
+        self.status_id: str = ''
+        self.activity: List[str] = []
+
+    def wait_for_completed(
```

Comment:
Maybe docstring for this method would be a nice place for example usage?

Also, any docstring for this method should be provided here as it is the main method we are going to use from this code.

Reply:
Moved usage example to class docstring